roles/glance/templates/etc/rsyncd.conf

```diff
@@ -6,7 +6,7 @@
     list = yes
     uid = glance
     gid = glance
-    hosts allow = {{ undercloud_cidr }}
+    hosts allow = {{ hostvars|ursula_controller_ips(groups)|join(', ') }}
```

Comment:
This also locks down rsync to only the controller nodes, rather than the entire "undercloud" network.